RxCocoa/iOS/DataSources/RxPickerViewAdapter.swift

```diff
@@ -31,12 +31,12 @@ private class RxPickerViewArrayDataSource<T>: NSObject, UIPickerViewDataSource,
     }
 }
 
-private class RxPickerViewSequenceDataSource<S: Sequence>
+public class RxPickerViewSequenceDataSource<S: Sequence>
```

Comment:
I don't think that anyone understands anymore what is by design and what works by pure accident here so I think it would be safer to also remove `private` from `RxPickerViewArrayDataSource`.